crates/cards/Cargo.toml

```diff
@@ -5,10 +5,13 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+default = ["serde"]
+
 [dependencies]
-time = { version = "0.3.20", features = ["serde", "serde-well-known", "std"] }
+time = { version = "0.3.20" }
 error-stack = "0.3.1"
-serde = { version = "1.0.155", features = ["derive"] }
+serde = { version = "1", features = ["derive"], optional = true }
 serde_json = "1.0.94"
```

Comment:
You can move `serde_json` to the `[dev-dependencies]` table since it's only used in tests.

Reply:
@divinenaman let me know if you'd need any help with this.